tools/spectral/ipa/rulesets/functions/IPA106ValidOperationID.js

```diff
@@ -0,0 +1,34 @@
+import { hasException } from './utils/exceptions.js';
+import { collectAdoption, collectAndReturnViolation } from './utils/collectionUtils.js';
+import { isCustomMethodIdentifier, getCustomMethodName, stripCustomMethodName } from './utils/resourceEvaluation.js';
+import { generateOperationID } from './utils/operationIdGeneration.js';
+
+const RULE_NAME = 'xgen-IPA-106-valid-operation-id';
+const ERROR_MESSAGE = 'Invalid OperationID';
```

Comment:
could we also include a link to the documentation where the developer can get more info to unblock themself?